env/work/srw_python/uti_plot.py

```diff
@@ -148,7 +148,8 @@ def uti_plot2d1d(ar2d, x_range, y_range, x=0, y=0, labels=('Horizontal Position'
     _backend.uti_plot2d1d(ar2d, x_range, y_range, x, y, labels, graphs_joined)
     
 #def uti_data_file_plot(_fname, _read_labels=1, _e=0, _x=0, _y=0, _graphs_joined=True):
-def uti_data_file_plot(_fname, _read_labels=1, _e=0, _x=0, _y=0, _graphs_joined=True, _traj_report=False, _traj_axis='x'): #MR29072016
+def uti_data_file_plot(_fname, _read_labels=1, _e=0, _x=0, _y=0, _graphs_joined=True, _traj_report=False, _traj_axis='x',
+                       log_scale=None, width_pixels=None): #MR29072016
```

Comment:
Consider using the same variable conventions that were previously used in this function, i.e. all names of its input parameters have "_" (underscore) prefix; consider adding it to "log_scale" and "width_pixels" (to have "_log_scale" and "_width_pixels").